src/test/java/seedu/geekeep/logic/LogicManagerTest.java

```diff
@@ -66,15 +66,45 @@
      */
     class TestDataHelper {
 
-        Task adam() throws Exception {
-            Title title = new Title("Adam Brown");
-            DateTime privateEndDateTime = new DateTime("01-05-17 1630");
+        Task event() throws Exception {
```

Comment:
Is it necessary though? We are mainly testing the dateTime over here. I think having the same title name as the method makes it clearer